mesosphere_signer.py

```diff
@@ -96,186 +104,204 @@
-    "asset": "ETHUSD",
-    "strPrice": "",
-    "timestamp": 0,
-    "lastPushedPrice": 0,
-    "timeLastPushed": 0
+	"requestId": 1,
+	"price": 0,
+	"asset": "ETHUSD",
+	"strPrice": "",
+	"timestamp": 0,
+	"lastPushedPrice": 0,
+	"timeLastPushed": 0
 }
 
 dai = {
-    "requestId": 39,
-    "price": 0,
-    "asset": "DAIUSD",
-    "strPrice": "",
-    "timestamp": 0,
-    "lastPushedPrice": 0,
-    "timeLastPushed": 0
+	"requestId": 39,
+	"price": 0,
+	"asset": "DAIUSD",
+	"strPrice": "",
+	"timestamp": 0,
+	"lastPushedPrice": 0,
+	"timeLastPushed": 0
 }
 
 eth_in_dai = {
-    "requestId": 1,
-    "price": 0,
-    "asset": "ETHDAI",
-    "strPrice": "",
-    "timestamp": 0,
-    "lastPushedPrice": 0,
-    "timeLastPushed": 0
+	"requestId": 1,
+	"price": 0,
+	"asset": "ETHDAI",
+	"strPrice": "",
+	"timestamp": 0,
+	"lastPushedPrice": 0,
+	"timeLastPushed": 0
 }
 
 # final data submission format
 
 submitData = {
-    # "starkKey":0,
-    "timestamp": 0,
-    "price": 0,
-    "assetName": 0,  # btc or eth
-    "oracleName": 0,
-    # "signatureR":0,
-    # "signatureS":0
+	# "starkKey":0,
+	"timestamp": 0,
+	"price": 0,
+	"assetName": 0,  # btc or eth
+	"oracleName": 0,
+	# "signatureR":0,
+	# "signatureS":0
 }
 
 
 def fetchAPI(public_api):
-    '''
-    Fetches price data from centralized public web API endpoints
-    Returns: (str) ticker price from public exchange web APIs
-    Input: (list of str) public api endpoint with any necessary json parsing keywords
-    '''
-    try:
-        # Parse list input
-        endpoint = public_api[0]
-        parsers = public_api[1:]
-
-        # Request JSON from public api endpoint
-        r = requests.get(endpoint)
-        json_ = r.json()
-
-        # Parse through json with pre-written keywords
-        for keyword in parsers:
-            json_ = json_[keyword]
-
-        # return price (last remaining element of the json)
-        price = json_
-        return float(price)
-    except BaseException:
-        response = 0
-        print('API ERROR', public_api[0])
+	'''
+	Fetches price data from centralized public web API endpoints
+	Returns: (str) ticker price from public exchange web APIs
+	Input: (list of str) public api endpoint with any necessary json parsing keywords
+	'''
+	try:
+		# Parse list input
+		endpoint = public_api[0]
+		parsers = public_api[1:]
+
+		# Request JSON from public api endpoint
+		r = requests.get(endpoint)
+		json_ = r.json()
+
+		# Parse through json with pre-written keywords
+		for keyword in parsers:
+			json_ = json_[keyword]
+
+		# return price (last remaining element of the json)
+		price = json_
+		return float(price)
+	except BaseException:
+		response = 0
+		print('API ERROR', public_api[0])
 
 
 def getAPIValues():
-    eth_in_dai["timestamp"] = int(time.time())
-    price = medianize(ethAPIs, daiAPIs)
-    eth_in_dai["price"] = int(price)
+	eth_in_dai["timestamp"] = int(time.time())
+	price = medianize(ethAPIs, daiAPIs)
+	eth_in_dai["price"] = int(price)
 
```

Comment:
It's updating the nonce, but not creating a new transaction with the incremented nonce. That's because the conditional is not satisfied on line 258. To fix, update the timestamp for WBTC in `getAPIValues`: `wbtc['timestamp'] = int(time.time())`.